gruntfile.js

```diff
@@ -537,4 +439,11 @@ module.exports = function (grunt) {
       }
     }
   });
+
+  grunt.registerTask('vendor', 'vendor:<min|reg>', function(min) {
+    var v = grunt.file.readYAML('vendor.yml');
+    for (type in v) { if (v.hasOwnProperty(type)) {
+      grunt.config('src.'+type+'Vendor',v[type][min]);
```

Comment:
I think that the default should be 'min'. That's the most used, because 'reg' is expected to be for development only.